src/components/JobCard/JobCard.tsx

```diff
@@ -0,0 +1,107 @@
+                                    <BusinessIcon fontSize="small" />
+                                    <Typography fontSize="medium" sx={{ ml: 1 }}>{company}</Typography>
+                                </Grid>
+                                <Grid item xs={12} sm={4} sx={{ display: 'flex', justifyContent: { xs: 'flex-start', sm: 'center' }, alignItems: 'center' }}>
+                                    <LocationOnIcon fontSize="small" />
+                                    <Typography fontSize="medium" sx={{ ml: 1 }}>{location}</Typography>
+                                </Grid>
+                                <Grid item xs={12} sm={4} sx={{ display: 'flex', justifyContent: { xs: 'flex-start', sm: 'center' }, alignItems: 'center' }}>
+                                    <AccountBalanceWalletIcon fontSize="small" />
+                                    <Typography fontSize="medium" sx={{ ml: 1 }}>{salary}</Typography>
+                                </Grid>
+                            </Grid>
+                            <Grid container spacing={1} mt={0.5} justifyContent="flex-start">
+                                {tags.map((tag, index) => (
+                                    <Grid item key={index}>
+                                        <Chip label={tag} sx={{ bgcolor: '#6A1B9A', color: 'white', height: 32, borderRadius: '8px' }} />
+                                    </Grid>
+                                ))}
+                            </Grid>
+                        </Grid>
+                        <Grid item xs={12} sm={12} md={2} sx={{ order: { xs: -1, sm: -1, md:0 }, alignSelf: { sm: 'flex-start' } }} display="flex">
+                            <Grid container spacing={1} justifyContent="right">
+                                <Grid item>
+                                    <AccessTimeIcon fontSize="small" />
+                                </Grid>
+                                <Grid item>
+                                    <Typography fontSize="medium">
+                                        { publicationDate }
+                                    </Typography>
+                                </Grid>
+                            </Grid>
+                        </Grid>
+                        <Grid container spacing={1} justifyContent="right">
+                            <Grid item>
+                                <Button
+                                    variant="contained"
+                                    color="secondary"
+                                    onClick={handleReadMoreClick}
+                                    sx={{
+                                        mt: 1,
+                                        bgcolor: '#6A1B9A',
+                                        color: 'white',
+                                        fontSize: 'small',
```

Comment:
If we don't set the color property, there is no need to set `bgcolor`. Additionally, let's keep the button at the default size otherwise it looks too much like the chip.

```suggestion
                                <Button
                                    variant="contained"
                                    onClick={handleReadMoreClick}
                                    sx={{
                                        mt: 1,
```